popUp.css

```diff
@@ -0,0 +1,194 @@
+#overlay {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  background: #ebf0ee;
+  width: 100vw;
+  height: 108vh;
+  opacity: 0.9;
+  z-index: 100;
+  display: none;
```

Comment:
- Your background color for the popup looks nice but make sure it sick as much as possible to the background shown in the Figma it is grey and a bit transparent so kindly fix that I provide below the color used in the Figma and to make it transparent I suggest you use mix-blend-mode it will make your design much more to the Figma design as it was required in the project :+1: 

```
background: #C1C7D0;
mix-blend-mode: multiply;
```

## your design 

![image](https://user-images.githubusercontent.com/95297251/204062515-87e4ac01-5b31-49ce-a992-6c29725518d9.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/204062479-d57bc135-6552-4f6d-9dc8-9a87ee718232.png)